packages/fxa-auth-server/lib/senders/emails/bin/mjml-sever.ts

```diff
@@ -0,0 +1,56 @@
+import http from 'http';
+import stream from 'stream';
+import net from 'net';
+import { render } from '../renderer';
+
+const { PORT = 8192, HOST = '0.0.0.0' } = process.env;
```

Comment:
I took the host and port from Les's repo so let me know if 8192 is already occupied or we're good to use it for development purpose

Reply:
It doesn't look like 8192 is occupied, but we _do_ have a pattern of using the 6000 range for Storybook. So this could be 6010.